app/config/collections.php

```diff
@@ -263,6 +263,17 @@
                 'array' => false,
                 'filters' => [],
             ],
+            [
+                '$id' => ID::custom('error'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 255,
```

Comment:
We might want to use a bigger limit here.

Reply:
2048 can be a good starting point.